### PR TITLE
adds formats registry considerations

### PR DESCRIPTION
**Pull request by baywet**
related #3196

### CODE REVIEW

**Review by ralfhandl — APPROVED** [review]

File: registry/format.md [hunk 1]
```diff
@@ -5,6 +5,16 @@ permalink: /registry/format/index.html
 
 # Formats Registry
 
+## Considerations
+
+The existence of a format in this registry DOES NOT require tools to implement it.
+
+If tools choose to implement any format present in this registry, they SHOULD implement the format following the provided standard.
```
Comment (MikeRalphson) [inline registry/format.md:12]:
```suggestion
If tools choose to implement any format present in this registry, they SHOULD implement the format following the provided behavior.
```

File: registry/format.md [hunk 2]
```diff
@@ -5,6 +5,16 @@ permalink: /registry/format/index.html
...
+
+Each format name and behavior MUST be unique to be added to this registry except when formats are ambiguous (no standard) by nature or deprecated.
```
Comment (MikeRalphson) [inline registry/format.md:14]:
```suggestion
Each format name and behavior MUST be unique to be added to this registry except when formats are ambiguous (no standard or single behavior) by nature or deprecated.
```

**Review by MikeRalphson — COMMENTED** [review]
LGTM - just a couple of suggestions as not all formats have a linked standard, but it doesn't necessarily make them ambiguous.

**Comment by baywet** [comment]
@handrews I just wanted to surface this one up for you in case you didn't see it :) Your feedback is always welcome.

**Review by handrews — COMMENTED** [review]
I'm generally OK with this, and don't feel super-strongly about the one comment I made.

File: registry/format.md [hunk 3]
```diff
@@ -5,6 +5,16 @@ permalink: /registry/format/index.html
 
 # Formats Registry
 
+## Considerations
+
+The existence of a format in this registry DOES NOT require tools to implement it.
+
+If tools choose to implement any format present in this registry, they SHOULD implement the format following the provided behavior.
+
+Each format name and behavior MUST be unique to be added to this registry except when formats are ambiguous (no standard or single behavior) by nature or deprecated.
```
Comment (darrelmiller) [inline registry/format.md:14]:
```suggestion
The registry SHOULD NOT contain two entries that have the same meaning, unless all but one have been deprecated.
```

File: registry/format.md [hunk 4]
```diff
@@ -5,6 +5,16 @@ permalink: /registry/format/index.html
...
+
+Ambiguous formats MUST contain recommendations to use more precise format definitions that exist in this registry or are part of the same submission.
```
Comment (darrelmiller) [inline registry/format.md:16]:
```suggestion
```
We don't believe this is necessary.

File: registry/format.md [hunk 5]
```diff
@@ -5,6 +5,16 @@ permalink: /registry/format/index.html
 
 # Formats Registry
 
+## Considerations
+
+The existence of a format in this registry DOES NOT require tools to implement it.
+
+If tools choose to implement any format present in this registry, they SHOULD implement the format following the provided behavior.
```
Comment (darrelmiller) [inline registry/format.md:12]: @webron  Please suggest a revision to this.

**Pull request merged**